src/region/RegionAnnotations.tsx

```diff
@@ -115,6 +115,7 @@ export default class RegionAnnotations extends React.PureComponent<Props, State>
                 {isCreating && staged && canReply && rectRef && (
                     <div className="ba-RegionAnnotations-popup">
                         <PopupReply
+                            data-resin-component="commentPopover"
```

Comment:
Can we sync with product on why our terminology differs from theirs? This form doesn't create a comment object.